grafana/containers/crud_details.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
Do we want new folder here or .container extension added to the file name ?
.common
.linux
.windows
.container
etc

Reply:
I would prefer to start organizing them in folders a bit if possible. We can still add extensions on to them as well.

Reply:
Me too but we are not their yet.
Can we do it in this release, yes it can be if we agree on folder organization.

---

Disclaimer: My vision is something like this
```
|---node_exporter
     |---- promethes/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- alert-manager/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- grafana/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---pg_exporter
     |---- promethes/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- alert-manager/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- grafana/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---blackbox_exporter
     |---- promethes/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- alert-manager/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- grafana/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---node_exporter
     |---- promethes/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- alert-manager/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
     |---- grafana/
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---prometheus
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---grafana
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
|---alert-manager
           |---- common/
           |---- linux/
           |---- windows/
           |---- containers/
```

Reply:
I like the common/linux/windows/containers subfolders, but do not want to split up each application's files across different folders like that. That is very confusing to me.

Reply:
Understand it is confusing but advantage is that it is self container per exporter, which will be easy to manage in long run as we add more configurations, platforms, exporters etc.

Reply:
Let's put this off until later vs trying to manage this full re-org in this PR. However this PR is adding a lot of dashboards dedicated to a specific purpose and filling up a single folder level where a distinction is clearly needed. Either way, we go, there is agreement on a containers folder in the Grafana folders, so let's just stick with that for now so this can move forward.

Reply:
We will have two different layouts as we merge this PR and this is what raised this concern.

Reply:
We're agreed on a folder hierarchy based on target platform in the grafana folder. If we go with the existing method of only going by file extension with everything in a single folder, then the process for the build/containers team to pull these files from the repository is going to be one way now then change later, possibly even in the very next release. Let's just take a small step now for better organization

Reply:
After initial implementation it will be hard to change.

Reply:
New issue created #189